accounts/tests/test_user.py

```diff
@@ -342,7 +344,7 @@ def test_user_delete_include_sounds_using_web_form(self, submit_job):
         self.assertRedirects(resp, reverse('front-page'))
 
         # Check loaded page contains message about user deletion
-        self.assertIn(resp.content, 'Your user account will be deleted in a few moments')
+        self.assertContains(resp, 'Your user account will be deleted in a few moments')
```

Comment:
https://docs.djangoproject.com/en/4.1/topics/testing/tools/#making-requests

> If you set follow to True the client will follow any redirects and a `redirect_chain` attribute will be set in the response object containing tuples of the intermediate urls and status codes.

One additional test we could do here is ensure that
* we are on the home page
* `resp.redirect_chain` includes the intermediate request to the delete page with the 302 redirect

Reply:
I believe the test in line 344 is exactly testing that.

Reply:
oh, excellent! perfect